src/tools/_tools.ms.scss

```diff
@@ -0,0 +1,14 @@
+// /*------------------------------------*\
+//     #MS
+// \*------------------------------------*/
+
+// To clearly indicate that a scale value is used, use it with a custom
+// function which just passes the number through.
+@function ms($scale: 0) {
+    // Check if `scales` list has `$scale` entry.
+    @if (index(map-get($FLUIDMS-CONFIG, scales), $scale)) {
```

Comment:
Use [`map-has-key`](http://www.sass-lang.com/documentation/modules/map#has-key) as this function is more descriptive and easier to read.

[See demo](https://codepen.io/akfzambrana/pen/LRgPXo)

Reply:
`scales` is a Sass **list** and thus the `map-has-key` function does not work.